Event.swift

```diff
@@ -36,7 +36,37 @@ class Event: PFObject, PFSubclassing {
     @NSManaged var amountPerAttendee: Int
     @NSManaged var location: String?
     @NSManaged var startTime: NSDate?
+    @NSManaged var endTime: NSDate?
+    var coverImageUrl: NSURL?
     
+    var moneyCollected: Int? {
+        return nil
+    }
+    
+    var totalMoneyToCollect: Int? {
+        return nil
+    }
```

Comment:
It's a preparation for the payment collection mechanism. I'm going to implement this further in the coming ticket.